core/json/src/main/scala/net/liftweb/json/JsonAST.scala

```diff
@@ -366,82 +522,107 @@ object JsonAST {
       append(this, other)
     }
 
-    /** Return a JSON where all fields matching the given predicate are removed.
-     * <p>
-     * Example:<pre>
+    /**
+     * Return a `JValue` where all fields matching the given predicate are removed.
+     *
+     * Example:
+     *
+     * {{{
      * JObject(("age", JInt(10)) :: Nil) removeField {
      *   case ("age", _) => true
      *   case _          => false
      * }
-     * </pre>
+     * }}}
      */
     def removeField(p: JField => Boolean): JValue = this mapField {
       case x if p(x) => JField(x.name, JNothing)
       case x => x
     }
 
-    /** Return a JSON where all values matching the given predicate are removed.
-     * <p>
-      * Example:<pre>
-      * JArray(JInt(1) :: JInt(2) :: JNull :: Nil) remove { _ == JNull }
-      * </pre>
-      */
+    /**
+     * Return a JSON where all values matching the given predicate are removed.
+     *
+     * Example:
+     *
+     * {{{
+     * JArray(JInt(1) :: JInt(2) :: JNull :: Nil).remove(_ == JNull)
+     * }}}
+     */
     def remove(p: JValue => Boolean): JValue = this map {
       case x if p(x) => JNothing
       case x => x
     }
 
-    /** Extract a value from a JSON.
-      * <p>
-      * Value can be:
-      * <ul>
-      *   <li>case class</li>
-      *   <li>primitive (String, Boolean, Date, etc.)</li>
-      *   <li>supported collection type (List, Seq, Map[String, _], Set)</li>
-      *   <li>any type which has a configured custom deserializer</li>
-      * </ul>
-      * <p>
-      * Example:<pre>
-      * case class Person(name: String)
-      * JObject(JField("name", JString("joe")) :: Nil).extract[Person] == Person("joe")
-      * </pre>
-      */
+    /**
+     * Extract a value into a concrete Scala instance from its `JValue` representation.
+     *
+     * Value can be:
+     *
+     * <ul>
```

Comment:
You can use:

```
  * Value can be:
  *  - a case class
  *  - a JSON primitive (`String`, `Boolean`, `Date`, etc)
  *  - supported ...
```

To avoid HTML here.